tests/autoyast/autoyast_verify.pm

```diff
@@ -92,7 +89,7 @@ sub test_flags {
     # 'fatal' - whole test suite is in danger if this fails
     # 'milestone' - after this test succeeds, update 'lastgood'
     # 'important' - if this fails, set the overall state to 'fail'
-    return {fatal => 1};
+    return {important => 1};
```

Comment:
maybe I'm thinking too much, but if this test is failed, worker will loaded lastgood, then all logs will be lost although you're tried to upload the logs at the last step :)